client/src/js/bundle.js

```diff
@@ -1,4 +1,3 @@
-import jQuery from 'jquery';
-import 'bootstrap/js/dist/popover';
+import Popover from 'bootstrap/js/dist/popover';
 
-jQuery('[data-toggle="popover"]').popover();
+[...document.querySelectorAll('[data-bs-toggle="popover"]')].map(popoverTriggerEl => new Popover(popoverTriggerEl));
```

Comment:
`popover()` function no longer exists - now using recommended code from https://getbootstrap.com/docs/5.3/components/popovers/#enable-popovers (but simplified to remove unnecessary variable assignments)